e2e/internal/kubeclient/deploy.go

```diff
@@ -350,6 +350,17 @@ func (c *Kubeclient) Restart(ctx context.Context, resource ResourceWaiter, names
 	return nil
 }
 
+// ScaleDeployment scales a deployment to the given number of replicas.
+func (c *Kubeclient) ScaleDeployment(ctx context.Context, namespace, name string, replicas int32) error {
+	deploy, err := c.client.AppsV1().Deployments(namespace).Get(ctx, name, metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+	deploy.Spec.Replicas = toPtr(replicas)
+	_, err = c.client.AppsV1().Deployments(namespace).Update(ctx, deploy, metav1.UpdateOptions{})
+	return err
+}
```

Comment:
Can we use [`DeploymentInterface.UpdateScale` or `DeploymentInterface.UpdateScale`](https://pkg.go.dev/k8s.io/client-go@v0.30.3/kubernetes/typed/apps/v1#DeploymentInterface)?

Reply:
Will do. btw: are both links the same or am I missing something?

Reply:
I just used one link. AFAICT there's no way to point to the specific method, so I just pointed to the interface.